core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -114,6 +126,14 @@ oppia.factory('threadDataService', [
       };
 
       $http.post(url, payload).success(function() {
+        if (updatedStatus != null) {
+          if (thread.status == 'open') {
```

Comment:
I suggest pulling 'open' out into a THREAD_STATUS_OPEN constant, since it's being used a few times.